Tests/LibCrypto/TestRSA.cpp

```diff
@@ -119,14 +120,19 @@ sV/ETwIDAQABAkBpC37UJkjWQRHyxP83xuasExuO6/mT5sQN692kcppTJ9wHNWoD
 RwIhAIDSm8Ajgf7m3RQEoLVrCe/l8WtCqsuWliOsr6rbQq4hAiEAx8R16wvOtZlN
 W4jvSU1+WwAaBZl21lfKf8OhLRXrmNkCIG9IRdcSiNR/Ut8QfD3N9Bb1HsUm+Bvz
 c8yGzl89pYST
------END PRIVATE KEY-----)"sv;
+-----END PRIVATE KEY-----
+)"sv;
     auto decoded = Crypto::decode_pem(keypem.bytes());
     auto keypair = Crypto::PK::RSA::parse_rsa_key(decoded);
     auto priv_der = MUST(keypair.private_key.export_as_der());
-    auto priv_pem = MUST(Crypto::encode_pem(priv_der, Crypto::PEMType::PrivateKey));
+    auto rsa_encryption_oid = Array<int, 7> { 1, 2, 840, 113549, 1, 1, 1 };
```

Comment:
we already have this as a constant in LibTLS/Certificate.h. ... perhaps it (and a few other parser items from LibTLS/Certificate.cpp?) should be moved to LibCrypto?

Reply:
other than that, this looks good, thanks!